emulators.tex

```diff
@@ -8,14 +8,127 @@ \chapter{Emulators}
+Xemu those are mainly operated with the six keys above the cursor keys. There you also
+find \specialkey{RUN STOP}, completely off in comparison to C64 emulator like VICE. The reason is, a Mega 65 even has an \specialkey{Esc} key. So some keys are approaching a
+more modern layout.
+
+\begin{center}
+  \includegraphics[width=\linewidth]{images/xemu-extended-keyboard.png}
+\end{center}
+
+As you might spot quickly with this setup \specialkey{RUN STOP} \widekey{RESTORE} could
+be done single-handed.
+
+Please be aware this image shows graphics symbols being reached in the orientation the
+\megasymbolkey and \specialkey{Shift} key are positioned on a real Mega 65. On a PC
+keyboard it is crossovered, unfortunately. So, while \megasymbolkey \megakey{S} gives
+you the upper right corner of a rectangle \specialkey{Shift} \megakey{S} gives you the
+heart symbol every new C64 owner must have tried first in the eightees.
+
+For the PC styled special character keys the lower symbol is reached without the use
+of modifier keys. The upper symbol shows what you get in conjunction with the
+\specialkey{Shift} key.
+
+Xemu lets you have two joysticks (one at a time) on the numeric keypad.
+\specialkey{Enter} over there is used to toggle ports 1, 2 and to off when pressed
+repeatedly.
+
+Please be aware the emulator by default catches \megakey{F9} \megakey{F10}
+\megakey{F11} as shortcuts of its own functions. If you want those keys to be used by
+the emulated Mega 65 you need to setup an own keyboard mapping. The files are found
+in the folder for Xemu's settings as explained on page
+\pageref{sec:sdcard-settings-location}. This is done by copying
+\stw{keymap-default.cfg} into \stw{keymap.cfg} and then removing the following lines:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+XEMU-EXIT F9
+XEMU-FULLSCREEN F11
+\end{verbatim}
+\end{tcolorbox}
+
+and changing:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+F9 Unknown
+F11 Unknown
+\end{verbatim}
+\end{tcolorbox}
+
+into:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+F9 F9
+F11 F11
+\end{verbatim}
+\end{tcolorbox}
+
+Fullscreen and exiting Xemu is still available in the menu you reach by right mouse
+clicking into the emu's main window.
+
+\subsection{Apple MacOS start problems}
+
+If a message \stw{macOS cannot verify that this app is free from malware} pops up you
+may want control-click to start it: \index{Apple} \index{macOS}
+
+\begin{itemize}
+  \item Open Finder on your Mac.
+  \item Point your mouse to Xemu's app icon.
+  \item Control-click it to open the shortcut menu.
+  \item Now click \stw{Open} to run Xemu. By doing it this way no warning will be shown.
+\end{itemize}
+
+\subsection{SD Card, settings location}
+\label{sec:sdcard-settings-location}
+\index{SD card}
+
+You may need to go to Xemu's settings folder in order to remap your keyboard if you want
```

Comment:
"You may wish to edit the Xemu settings file to change the keyboard mapping or set the default SD card image file. To access this file from within Xemu, open the menu, select "Debug / Advanced," then "Browse system folder." Edit the \texttt{keymap-default.cfg} and \texttt{mega65-default.cfg} files with a text editor."

Reply:
There is no way to *set a default SD card* or we should explain how this could be obtained and what it could be good for. I think Xemu expects to have partitioned the sparse file all by itself.

Your description on how to edit the keymap is redundant to an above section handling exact those two files and showing the keyboard layout. However, I updated all using your text.

Reply:
That's a good point about the redundancy. Repetition isn't always bad, but please feel free to rework my suggestion to fit the chapter.

Reply:
Please find an edited version to review.